GDAXClient/Services/Currency.cs

```diff
@@ -4,6 +4,7 @@ public enum Currency
     {
         USD,
         EUR,
-        GBP
+        GBP,
+        BTC
```

Comment:
there's an enum called `ProductType` that includes BTC. do you think it would it make more sense to use that here instead of adding as a currency?